webapp/app/js/models/resultsets.js

```diff
@@ -41,9 +41,10 @@ treeherder.factory('ThResultSetModel', [
     var pollDelayMin = 1000;
     var pollDelayMax = 60000;
 
-    // if any of these params are in the url, don't poll for new data
-    var noPollingParameters = [
-        'fromchange', 'tochange', 'startdate', 'enddate', 'revision'
+    // Keys that, if present on the url, must be passed into the resultset
+    // polling endpoint
+    var rsPollingKeys = [
+        'tochange', 'enddate', 'revision', 'author'
     ];
```

Comment:
this is one I can often go either way on.  But, yeah.  Probably defaulting to a single line where possible is better.  :)